tutorials/toolkits/axisartist.py

```diff
@@ -1,6 +1,153 @@
-.. _axisartist-manual:
+"""
+==============================
+Overview of axisartist toolkit
+==============================
+
+The axisartist toolkit tutorial.
+
+.. warning::
+   *axisartist* uses a custom Axes class
+   (derived from the mpl's original Axes class).
+   As a side effect, some commands (mostly tick-related) do not work.
+
+The *axisartist* contains custom Axes class that is meant to support for
+curvilinear grids (e.g., the world coordinate system in astronomy).
+Unlike mpl's original Axes class which uses Axes.xaxis and Axes.yaxis
+to draw ticks, ticklines and etc., Axes in axisartist uses special
+artist (AxisArtist) which can handle tick, ticklines and etc. for
+curved coordinate systems.
+
+.. figure:: ../../gallery/axisartist/images/sphx_glr_demo_floating_axis_001.png
+   :target: ../../gallery/axisartist/demo_floating_axis.html
+   :align: center
+   :scale: 50
+
+   Demo Floating Axis
+
+Since it uses special artists, some mpl commands that work on
+Axes.xaxis and Axes.yaxis may not work.
+
+.. _axisartist_users-guide-index:
+
+axisartist
+==========
+
+*axisartist* module provides a custom (and very experimental) Axes
+class, where each axis (left, right, top and bottom) have a separate
+associated artist which is responsible to draw axis-line, ticks,
+ticklabels, label.  Also, you can create your own axis, which can pass
```

Comment:
This is just unclear...what is passing through a fixed postion, and passing through to what?